Tests/conf.json

```diff
@@ -3157,6 +3157,21 @@
             "playbookID": "Google Chronicle Backstory List Events - Test",
             "fromversion": "5.0.0"
         },
+        {
+            "integrations": "Google Chronicle Backstory",
+            "playbookID": "playbook-GoogleChronicleBackstoryRetrohunts-Test",
+            "fromversion": "5.0.0"
+        },
+        {
+            "integrations": "Google Chronicle Backstory",
+            "playbookID": "playbook-GoogleChronicleBackstoryRules-Test",
+            "fromversion": "5.0.0"
+        },
+        {
+            "integrations": "Google Chronicle Backstory",
+            "playbookID": "playbook-GoogleChronicleBackstoryReferenceList-Test",
+            "fromversion": "5.0.0"
+        },
```

Comment:
No need to add these to conf.json, we don't run partner test PB as part of our internal build anymore.

Reply:
Ok, Removed these entries from conf.json.